kakao/src/main/kotlin/com/agoda/kakao/common/matchers/ToolbarTitleMatcher.kt

```diff
@@ -27,6 +27,6 @@ class ToolbarTitleMatcher() : BoundedMatcher<View, Toolbar>(Toolbar::class.java)
         if (title == null && resId != null) {
             title = resId?.let { view?.context?.getString(it) }
         }
-        return view?.title?.equals(title) ?: false
+        return view?.title?.toString().equals(title)
```

Comment:
`?: false` should not be removed, or else return type of the function becomes `Boolean?`

Reply:
And you can replace `.equals(title)` to `== title` in this case

Reply:
👍  `== title` also works if `view?.title?` is `null` - better solution. fixed that.